meet-data/nasa/todo/getpdfs.sh

```diff
@@ -3,16 +3,16 @@
 #One by one, it fetches the pdf, creates a new folder, moves the pdf to that folder, applies tabula to it
 #runs nasa-standardize to it, applies check-entries and stores the output.
 input="pdf_small"
-COUNTER=10
+COUNTER=1
 while IFS= read -r var
 do
-  mkdir /Users/blerner/openpowerlifting/meet-data/nasa/todo/"$COUNTER"
+  mkdir /Users/blerner/openpowerlifting/meet-data/nasa//140"$COUNTER"
 	wget $var
-	mv *.pdf /Users/blerner/openpowerlifting/meet-data/nasa/todo/"$COUNTER"
-	cd /Users/blerner/openpowerlifting/meet-data/nasa/todo/"$COUNTER"
+	mv *.pdf /Users/blerner/openpowerlifting/meet-data/nasa/140"$COUNTER"
+	cd /Users/blerner/openpowerlifting/meet-data/nasa/140"$COUNTER"
 	for f in *;
 	do
-		java -jar /Users/blerner/openpowerlifting/tabula-1.0.1-jar-with-dependencies.jar -l $f >"results.csv"
+		java -jar /Users/blerner/openpowerlifting/tabula-1.0.1-jar-with-dependencies.jar -l -p all $f >"results.csv"
```

Comment:
My Java does not have a `-p` option. On my system, this will fail. What does it do on your system? Can you maybe make it MacOS-only?

Granted, this is not such a big deal, since your username is hardcoded all over the place above :-)

Reply:
the -p is very importnat for me. It stands for "pages". Without -p all only the first page gets done, which happened in the past